integration/tests/managed/managed_nodegroup_test.go

```diff
@@ -161,142 +162,14 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
-
-			By("correctly configuring the bottlerocket nodegroup")
-			kubeTest, err := kube.NewTest(params.KubeconfigPath)
-			Expect(err).NotTo(HaveOccurred())
-
-			nodeList := kubeTest.ListNodes(metav1.ListOptions{
-				LabelSelector: fmt.Sprintf("%s=%s", "eks.amazonaws.com/nodegroup", bottlerocketNodegroup),
-			})
-			Expect(nodeList.Items).NotTo(BeEmpty())
-			for _, node := range nodeList.Items {
-				Expect(node.Status.NodeInfo.OSImage).To(ContainSubstring("Bottlerocket"))
-				Expect(node.Labels["kubernetes.io/hostname"]).To(Equal("custom-bottlerocket-host"))
-			}
-			kubeTest.Close()
-		})
-
-		It("should have created an EKS cluster and 4 CloudFormation stacks", func() {
-			awsSession := NewConfig(params.Region)
-
-			Expect(awsSession).To(HaveExistingCluster(params.ClusterName, string(ekstypes.ClusterStatusActive), params.Version))
-
-			Expect(awsSession).To(HaveExistingStack(fmt.Sprintf("eksctl-%s-cluster", params.ClusterName)))
-			Expect(awsSession).To(HaveExistingStack(fmt.Sprintf("eksctl-%s-nodegroup-%s", params.ClusterName, initialAl2Nodegroup)))
-			Expect(awsSession).To(HaveExistingStack(fmt.Sprintf("eksctl-%s-nodegroup-%s", params.ClusterName, bottlerocketNodegroup)))
-			Expect(awsSession).To(HaveExistingStack(fmt.Sprintf("eksctl-%s-nodegroup-%s", params.ClusterName, ubuntuNodegroup)))
-		})
-
-		It("should have created a valid kubectl config file", func() {
-			config, err := clientcmd.LoadFromFile(params.KubeconfigPath)
-			Expect(err).ShouldNot(HaveOccurred())
-
-			err = clientcmd.ConfirmUsable(*config, "")
-			Expect(err).ShouldNot(HaveOccurred())
-
-			Expect(config.CurrentContext).To(ContainSubstring("eksctl"))
-			Expect(config.CurrentContext).To(ContainSubstring(params.ClusterName))
-			Expect(config.CurrentContext).To(ContainSubstring(params.Region))
-		})
-
-		Context("and listing clusters", func() {
-			It("should return the previously created cluster", func() {
-				cmd := params.EksctlGetCmd.WithArgs("clusters", "--all-regions")
-				Expect(cmd).To(RunSuccessfullyWithOutputString(ContainSubstring(params.ClusterName)))
-			})
-		})
-
-		Context("and checking the nodegroup health", func() {
-			It("should return healthy", func() {
-				checkNg := func(ngName string) {
-					cmd := params.EksctlUtilsCmd.WithArgs(
-						"nodegroup-health",
-						"--cluster", params.ClusterName,
-						"--name", ngName,
-					)
-
-					Expect(cmd).To(RunSuccessfullyWithOutputString(ContainSubstring("active")))
-				}
-
-				checkNg(initialAl2Nodegroup)
-				checkNg(bottlerocketNodegroup)
-				checkNg(ubuntuNodegroup)
-			})
-		})
-
-		Context("and scale the initial nodegroup", func() {
-			It("should not return an error", func() {
-				cmd := params.EksctlScaleNodeGroupCmd.WithArgs(
-					"--cluster", params.ClusterName,
-					"--nodes-min", "2",
-					"--nodes", "3",
-					"--nodes-max", "4",
-					"--name", initialAl2Nodegroup,
-				)
-				Expect(cmd).To(RunSuccessfully())
-			})
-		})
-
```

Comment:
Do the tests fail if this code isn't moved around?

Reply:
No. I moved it around to speed up the tests mostly.